docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md

```diff
@@ -38,7 +38,7 @@ If the output of the above is blank, you should proceed with installing the driv
 ```bash
 sudo apt update
 sudo apt -y install dkms
-wget https://download.01.org/intel-sgx/sgx-dcap/1.7/linux/distro/ubuntu18.04-server/sgx_linux_x64_driver_1.35.bin -O sgx_linux_x64_driver.bin
+wget https://download.01.org/intel-sgx/sgx-dcap/1.10/linux/distro/ubuntu18.04-server/sgx_linux_x64_driver_1.41.bin -O sgx_linux_x64_driver.bin
 chmod +x sgx_linux_x64_driver.bin
```

Comment:
@gkostal 1.41 has a breaking change and we are still on 1.35. Can we revert this particular line

Reply:
Thanks.  I was just following the directions to get the latest from Intel's SGX site.  If this isn't accurate, it should probably be changed.

![image](https://user-images.githubusercontent.com/7255232/109717574-15a04f00-7b5b-11eb-8235-c9e562d02118.png)

Reply:
That’s totally valid. Thanks